apps/web/src/components/FormContent.tsx

```diff
@@ -0,0 +1,39 @@
+import { FormStepHeader }  from '@components';
+import {Profile}  from '../pages/forms/Profile';
+import {CareActivitiesBundle} from '../pages/forms/CareActivitiesBundle';
+import {Occupation} from '../pages/forms/Occupation';
+import {ActivitiesGap} from '../pages/forms/ActivitiesGap';
+import {Suggestions} from '../pages/forms/Suggestions';
+
+interface FormContentProps {
+  step: number;
+  formTitle: string;
+}
+
+export const FormContent: React.FC<FormContentProps> = ({step, formTitle}) => {
+  const stepCount = step;
+
+  const showStepContent =()=> {
+    switch (String(step)) {
+      case "1": 
```

Comment:
I meant Profile, CareActivities and stuffs. Numbers can be confusing in the long run

Reply:
Lol, k . Changed